gui/src/app/pages/HomePage/HomePage.tsx

```diff
@@ -44,6 +47,18 @@ const HomePage: FunctionComponent<Props> = ({ width, height }) => {
         localStorage.setItem('data.json', dataFileContent)
     }, [dataFileContent])
 
+    const [samplingOptsContent, setSamplingOptsContent] = useState(initialSamplingOptsContent)
+    useEffect(() => {
+        localStorage.setItem('samplingOpts.json', samplingOptsContent)
```

Comment:
I assume setting this in local storage is to make it persistent between refreshes? I'm not sure that's the best behavior--it seems kind of weird that I can't get the defaults back by refreshing the page.

Reply:
Yeah I here what you are saying. We can work on this in a separate issue.